lib/std/meta.zig

```diff
@@ -991,20 +1041,58 @@ pub fn sizeof(target: anytype) usize {
             // TODO test if target fits in int, long or long long
             return @sizeOf(c_int);
         },
-        else => @compileError("TODO implement std.meta.sizeof for type " ++ @typeName(@TypeOf(target))),
+        .EnumLiteral => {
```

Comment:
`translate-c` should never generate enum literals so I don't think we should allow them here, same goes for `.Vector`.

Reply:
Makes sense for enum literal but it seems like we might eventually want to translate __m128 and friends to vector.

Reply:
I guess that could happen, I was mostly going with what already exists.

Reply:
I removed the .EnumLiteral case, but it seems plausible that .Vector will be used at some point, and that whoever does that update to translate-c will not remember to modify this function.  I don't really see a problem with leaving it in, but if you feel strongly about it I'll take it out.